File: sBuilding/Trunk/Server/Java/meetingservice/src/com/swt/meeting/domain/PersonNotBarcode.java
package com.swt.meeting.domain;

import java.io.Serializable;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "swt_smeeting_person_not_barcode")
public class PersonNotBarcode implements Serializable{
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;

	@Id
	@GeneratedValue
	private long id;

	@Column(name = "name", length = 50)
	private String name;

	@Column(name = "orgname", length = 150)
	private String orgName;
	@Column(name = "position")
	private String position;
	@Column(name = "identitycard", length = 30)
	private String identityCard;

	@Column(name = "phonenumber", length = 25)
	private String phonenumber;

	public long getId() {
		return id;
	}

	public void setId(long id) {
		this.id = id;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getOrgName() {
		return orgName;
	}

	public void setOrgName(String orgName) {
		this.orgName = orgName;
	}

	public String getPosition() {
		return position;
	}

	public void setPosition(String position) {
		this.position = position;
	}

	public String getIdentityCard() {
		return identityCard;
	}

	public void setIdentityCard(String identityCard) {
		this.identityCard = identityCard;
	}

	public String getPhonenumber() {
		return phonenumber;
	}

	public void setPhonenumber(String phonenumber) {
		this.phonenumber = phonenumber;
	}

}

File: sBuilding/Trunk/Client/sBuilding/sMeetingComponent/Model/CustomObj/PersonNotBarcode/PersonNotBarcode.cs
using System;
using System.Runtime.Serialization;

namespace sMeetingComponent.Model.CustomObj.PersonNotBarcode
{
    [DataContract]
    public class PersonNotBarcode
    {
        public PersonNotBarcode()
        { }
        [DataMember]
        public long id { get; set; }
        [DataMember]
        public String name { get; set; }
        [DataMember]
        public String orgName { get; set; }
        [DataMember]
        public String position { get; set; }
        [DataMember]
        public String identityCard { get; set; }
        [DataMember]
        public String phonenumber { get; set; }
    }
}
